teamA/jihyun/db/migrations/20220803134924_create_posts_table.sql

```diff
@@ -1,11 +1,14 @@
 -- migrate:up
 CREATE TABLE posts (
     id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    user_id INT NULL,
     title VARCHAR(100) NOT NULL,
     content VARCHAR(2000) NULL,
     created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    CONSTRAINT posts_ibfk_1 FOREIGN KEY (user_id) REFERENCES users(id)
```

Comment:
제약조건 설정은 아래와 같이 테이블에 컬럼명을 조합해서 만들어주세요.

```suggestion
    CONSTRAINT posts_user_id FOREIGN KEY (user_id) REFERENCES users(id)
```